python/baseline/tf/remote.py

```diff
@@ -1,6 +1,85 @@
 import numpy as np
 from baseline.utils import import_user_module
-import tensorflow as tf
+
+
+class RemoteModelTensorFlowREST(object):
+
+    def __init__(self, remote, name, signature, labels=None, beam=None, lengths_key=None, inputs=[]):
+
+        self.remote = remote
+        self.name = name
+        self.signature = signature
+        self.lengths_key = lengths_key
+        self.input_keys = set(inputs)
+        self.beam = beam
+        self.labels = labels
+
+    def get_labels(self):
+        return self.labels
+
+    def predict(self, examples):
+        import requests
+
+        valid_example = all(k in examples for k in self.input_keys)
+        if not valid_example:
+            raise ValueError("should have keys: " + ",".join(self.input_keys))
+
+        url = '{}/v1/models/{}/versions/1:predict'.format(self.remote, self.name)
+        request = self.create_request(examples)
+        outcomes_list = requests.post(url, json=request)
+        outcomes_list = outcomes_list.json()['outputs']
+        outcomes_list = self.deserialize_response(examples, outcomes_list)
+        return outcomes_list
+
+    def deserialize_response(self, examples, predict_response):
```

Comment:
there could be a helper method here shared between rest and grpc for assembling classes and scores.

Reply:
maybe, but I dont think it will really eliminate any code, and it may cause the code to be harder to read